src/CMakeLists.txt

```diff
@@ -55,11 +56,12 @@ target_include_directories(benchmark PUBLIC
     )
 target_link_libraries(benchmark_main benchmark)
 
-set(include_install_dir "include")
-set(lib_install_dir "lib/")
-set(bin_install_dir "bin/")
-set(config_install_dir "lib/cmake/${PROJECT_NAME}")
-set(pkgconfig_install_dir "lib/pkgconfig")
+#set(include_install_dir "${CMAKE_INSTALL_INCLUDEDIR}")
+#set(lib_install_dir "${CMAKE_INSTALL_LIBDIR}")
+#set(bin_install_dir "${CMAKE_INSTALL_BINDIR}")
+set(config_install_dir "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
```

Comment:
Please just completely drop all these 5 original variables, and replace them with whatever their value would be

Reply:
do these not end up being sensibly defaulted to the CMAKE_ versions in this case?

Reply:
There is no point in having these immediate variables, they are just confusing.